khiva/clustering.py

```diff
@@ -25,6 +25,8 @@ def k_means(tss, k, tolerance=1e-10, max_iterations=100):
     :param k:                   The number of means to be computed.
     :param tolerance:           The error tolerance to stop the computation of the centroids.
     :param max_iterations:      The maximum number of iterations allowed.
+
+    :return: Array of centroids and Array of Labels.
```

Comment:
Tuple with a KHIVA array containing the centroids and a KHIVA array containing the labels. (Please be consistent with the upper cases..Labels)